.gitignore

```diff
@@ -37,3 +37,11 @@ install/kubernetes/istio-one-namespace.yaml
 install/kubernetes/istio.yaml
 samples/bookinfo/consul/bookinfo.sidecars.yaml
 samples/bookinfo/eureka/bookinfo.sidecars.yaml
+/bin/istio_ca
```

Comment:
starting / seems odd ?

Reply:
that's to avoid matching nested directories

Reply:
Please don't build istio binaries in the source directory !

( we're still in process of removing the envoy binary and all the docker ones )